programs/validator-history/src/utils.rs

```diff
@@ -1,13 +1,21 @@
 use anchor_lang::{
-    prelude::{AccountInfo, Pubkey},
+    prelude::{AccountInfo, Pubkey, Result},
+    require,
     solana_program::native_token::lamports_to_sol,
 };
 
-pub fn cast_epoch(epoch: u64) -> u16 {
-    (epoch % u16::MAX as u64).try_into().unwrap()
+use crate::errors::ValidatorHistoryError;
+
+pub fn cast_epoch(epoch: u64) -> Result<u16> {
```

Comment:
result not used. require panics + unwrap too.

Reply:
`require` returns an `Err` if the condition is hit, so the anchor Result is needed here

Reply:
I think it's a macro that expands into something that returns Err so the ? is not needed. If you have require in a function that doesn't return Result<> it complains